daisy/resource_registry.go

```diff
@@ -89,13 +89,14 @@ func (r *baseResourceRegistry) start(name string) DError {
 		return Errf("cannot start %s %q; does not exist in registry", r.typeName, name)
 	}
 
-	if !res.stopped {
+	if res.started {
```

Comment:
Whats the reason behind this change? What information does this give us that 'stopped' does not?

Reply:
I did this change because the original code doesn't allow a instance to be started without a pre 'stop' step. However, when we operate an instance, it can be either stopped or started.

Reply:
ok, is there a way to simply change this to just use started? just wondering if we can consolidate for readability

Reply:
no  - if we use only "started", then we are not able to stop a instacne without a pre "start" step. One single bool can only represent 2 scenarios, while there are at least 3 scenarios:
1. instance has been started by a pre step.
2. instance has been stopped by a pre step.
3. instance is not started/stopped by any pre step.

For readability, let me rename the var name to "started_by_wf" and "stopped_by_wf".